app/models/classification.rb

```diff
@@ -492,8 +492,8 @@ def validate_format_of_name
     end
   end
 
-  def self.name2tag(name, parent_id = 0, ns = DEFAULT_NAMESPACE)
-    if parent_id == 0
+  def self.name2tag(name, parent_id = nil, ns = DEFAULT_NAMESPACE)
+    if parent_id == 0 || parent_id.nil?
```

Comment:
I expected you'd either support {0, nil} everywhere or only {nil} everywhere?
Omission, or deliberate difference from is_entry, category? etc?

Reply:
I was concerned about this one. `name2tag` is called from just so many places. concerned one will sneak through.

Reply:
I'm also concerned when we import old `yaml` files that have a parent_id of `0` instead of the new way `null`